client/src/views/Help.vue

```diff
@@ -0,0 +1,78 @@
+                    <br>If the player has no matches or they choose not to play any of their cards even though they might have a match, they must draw a card from the Draw pile. If that card can be played, play it. Otherwise, keep the card, and the game moves on to the next person in turn. You can also play a Wild card, or a Wild Draw Four card on your turn.<br>
+                    <br>Note: If the first card turned up from the Draw Pile (to form the Discard Pile) is an Action card, the Action from that card applies and must be carried out by the first player to go (as stated, it is usually the player to the dealer’s left). The exceptions are if a Wild or Wild Draw Four card is turned up.
+                    If it is a Wild card, the first player to start (usually the one on the dealer’s left), can choose whatever color to begin play. If the first card is a Wild Draw Four card – Return it to the Draw Pile, shuffle the deck, and turn over a new card.<br>
+                    <br>Take note that you can only put down one card at a time; you cannot stack two or more cards together on the same turn. For example, you cannot put down a Draw Two on top of another Draw Two, or Wild Draw Four during the same turn, or put down two Wild Draw Four cards together.<br>
+                    <br>The game continues until a player has one card left. The moment a player has just one card they must yell “UNO!”. If they are caught not saying “Uno” by another player before the next player has taken their turn, that player must draw two new cards as a penalty. Assuming that the player is unable to play/discard their last card and needs to draw, but after drawing, is then able to play/discard that penultimate card, the player has to repeat the action of calling out “Uno”. The bottom line is – Announcing “Uno” needs to be repeated every time you are left with one card.
+                    Once a player has no cards remaining, the game round is over, points are scored, and the game begins over again. Normally, everyone tries to be the first one to achieve 500 points, but you can also choose whatever points number to win the game, as long as everyone agrees to it.<br>
+                </p></a>
+            </section>    
+            <section id="tutorials"> 
+                              
+                <h2> Tutorials: </h2>
+                <a id="section2"><p class=text > Tutorials here</p></a>
+            </section>    
+            <section id="cardAbilities">
+                
+                <h2> Card Abilities: </h2>
+            
+                <a id="section3"><p class=text  ><br><strong>Action Cards:</strong>  Besides the number cards, there are several other cards that help mix up the game. These are called Action or Symbol cards.<br>
+
+                    <br><strong>Uno Action Cards</strong><br>
+
+                    <Card :number="1" :color="'red'"></Card><br><br>
```

Comment:
These should reflect the actual cards. Right now they are all just Red-1's

Reply:
I didn't make this part so I didn't even notice haha, I just fixed it